apps/web/src/assets/css/dashboard.scss

```
@@ -40,33 +40,16 @@
   }
 }
 
-.audit-log-layout {
-  display: flex;
-  gap: 0;
+// The audit-log filter/results layout is now provided by the MOJ filter-layout
+// object. Only these two content helpers remain: they style cells inside the
+// results table, not the filter panel.
+.audit-log-layout__word-break {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)